mira/sources/util.py

```diff
@@ -1,34 +1,33 @@
-__all__ = ['transition_to_templates', 'get_sympy']
+__all__ = ['transition_to_templates', 'get_sympy', 'parameter_to_mira',
+           'safe_parse_expr']
```

Comment:
I wouldn't put `safe_parse_expr` here since it's not defined in this module. We should import it from the `metamodel` module in any code that uses it.